src/components/AnnotationList/AnnotationList.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../variables.scss';
+
+.ba .ba-annotation-list {
+    padding-bottom: 10px;
+    width: 100%;
+
+    .ba-annotation-list-item {
+        list-style: none;
+
+        // Spaces annotation elements from each other and the dividing border
+        // Adds extra spacing above consecutive annotations as well
+        margin-top: 23px; // Matches spacing of activity feed items in UI Elements
```

Comment:
Are these duplicate styles? I thought I saw them in a previous PR.

Reply:
They'll get deleted from Annotator.scss and into this component specific stylesheet when I integrate this component into the Dialog

Reply:
Removed it from Annotator.scss